ci/test.yml

```diff
@@ -362,6 +362,7 @@ core hwi test:
   <<: *gitlab_caching
   needs:
     - core unix frozen debug build
+  allow_failure: true
```

Comment:
There are also `legacy` `HWI` tests which should be allowed to fail

Reply:
added in rebase